src/entry-point-client/index.tsx

```diff
@@ -1,3 +1,5 @@
+import "regenerator-runtime/runtime";
```

Comment:
good questions. 
1. for some reason adding the `camelcase` dependency started triggering this error. Possibly some difference in the way it was transpiled? I suspect we probably did need it all along for older browser support but just haven't triggered an error thus far. 
2. including this here covers static pages as well because we load it from `index.html`

Reply:
oh ok both of those make sense, thanks for investigating these!